packages/vsphere-cpi/charts/questions.yaml

```diff
@@ -0,0 +1,44 @@
+rancher_min_version: 2.5.6-rc1
+rancher_max_version: 2.5.6-rc99
```

Comment:
I'm not familiar with putting this inside questions.yml, what is goal for this ?

Reply:
@PennyScissors can chime in but I believe we added this to block releasing this Helm chart to users before the release was about to happen.

Do we need any limits on rancher_min_version and rancher_max_version for this chart as a feature chart? I think the answer is no, but I want to confirm.

Reply:
This is to hide the charts because helm3-charts is an external facing repo and QA needs to validate them first. AFAIK, feature charts don't use this, so if that's the case we can just remove it. 

@deniseschannon You mentioned you wanted the vsphere charts to be available for v2.5.6+, so I figured you should be aware of this.

Reply:
Apparently this doesn't work in vue at all, so lets just remove. We haven't added min/max to feature charts yet in general

Reply:
Updated in latest commit.